internal/supervisor/translation.go

```diff
@@ -100,6 +105,36 @@ func (s *TranslationSupervisor) supervise() {
 		logger.WithError(err).Error("Failed to mark translation as completed")
 		return
 	}
+	defer func() {
+		if err := trans.Cleanup(); err != nil {
+			logger.WithError(err).Error("error cleaning up translation")
+		}
+	}()
+
+	// Only validate if the origin is not a mattermost type, since we validate those on the API calls
+	if translation.Type != model.MattermostWorkspaceBackupType {
+		logger.Info("Valiating translation result")
+		// Validate the translation before considering it "importable"
+		validator, err := validators.NewValidator(model.MattermostWorkspaceBackupType)
```

Comment:
I think we need `validators.NewValidator(translation.Type)`

Reply:
It is forced to a Mattermost type because the translation generates a mattermost type archive, so we need to validate always that.